meta-resin-common/recipes-core/initrdscripts/files/fsck

```diff
@@ -6,6 +6,12 @@ fsck_enabled() {
         echo "[INFO] Flasher detected. We don't run fsck on the partitions when provisioning."
         return 1
     fi
+
```

Comment:
Add implement this as a file to be slightly easier to maintain.

Reply:
I prefer this way which makes it obvious to anyone who reads the script later. I'm hoping we never need to add any other options to this file.